index.html

```diff
@@ -2,11 +2,151 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Седона</title>
   </head>
   <body>
+    <header class="page-header">
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">
+              <img class="logo" src="#" alt="логотип Седоны">
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">О Седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">Гостиницы</a>
+          </li>
+        </ul>
+        <ul class="navigation-user">
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">
+              <span class="visually-hidden">
+              Поиск
+              </span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a href="#" class="navigation-link">
+             <span class="visually-hidden">
+              Избранное
+             </span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a href="#" class="button">Хочу сюда!</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-index">
+      <h1 class="visually-hidden">Город Седона</h1>
+      <img src="#" class="Welcome-sedona" alt="Welcome to the gorgeous Sedona. Because the Grand Canyon sucks!">
```

Comment:
```suggestion
      <div>
            <img src="#" class="Welcome-sedona" alt="Welcome to the gorgeous Sedona. Because the Grand Canyon sucks!">
      </div>
```